configs/babel.config.js

```diff
@@ -7,6 +7,11 @@ module.exports = {
         [
             "@babel/env",
             {
+                "useBuiltIns": "usage",
```

Comment:
I do not know `"useBuiltIns": "usage"` stability.
For example, CRA does not use it.
https://github.com/facebook/create-react-app/blob/f5c3bdb65480f93b2d4a4c2f3214fc50753de434/packages/babel-preset-react-app/create.js#L80-L90

I want to reduce babel specific bahavior. It increase maintain cost.
In the future, we may use another transpiler.
So, I have lessed configuration.

Reply:
Probably CRA doesn't use it only because CRA is for application developers, so it just use `usage: entry` to include all polyfill for target web-browsers. But textlint rule is not a application, so no need to include all polyfill.